### DISCUSSION TITLE
Intellisense

### DISCUSSION THREAD

**Discussion by mcrascal** [evidence-dev/evidence#2374] [General]

**Comment by hughess** [comment]
Alternative approaches discussed here in the VS Code extension repo: https://github.com/evidence-dev/evidence-vscode/issues/120

**Comment by mcrascal** [comment]
@csjh has a rough POC here: https://github.com/csjh/language-tools/pull/1/files

Current thinking is we'll publish the above as a language server extension, and make that a dependency of the vs code extension. 

Potentially use the isEvidenceProject flag in the existing VS code extension to prevent this from firing on non-evidence markdown files. 

This has a fair amount of jank. Unclear how well this works inside a monorepo structure etc.
